x/examples/fyne-tools/README.md

```diff
@@ -14,6 +16,31 @@ go run fyne.io/fyne/v2/cmd/fyne package -os android && adb install Net_Tools.apk
 Note: the generated APK is around 85MB.
 
 
+## Windows
+
+From macOS, you can build the app for Windows with MinGW.
+
+First install MinGW:
+
+```
+brew install mingw-w64
+```
+
+Build for 64-bit:
+
+```
+GOOS=windows GOARCH=amd64 CGO_ENABLED=1 CC="x86_64-w64-mingw32-gcc" go build .
+```
+
+32-bit:
```

Comment:
I'm thinking whether we should still put 32-bit build instructions here, are there any real world PCs that are still 32 bit?

Reply:
Agree. removed the 32-bit instructions.